include/service/entities/article_service.js

```diff
@@ -191,6 +191,28 @@ ArticleService.prototype.find = function(where, options, cb) {
 };
 
 /**
+ * Updates articles
+ *
+ * @param {String} articleId	id of article
+ * @param {Object} options	
+ * @param {Object} fields	fields to update	
+ * @param {Function} cb      Callback function
+ */
+ArticleService.prototype.update = function(articleId, options, fields, cb) {
```

Comment:
switch the order of options and fields.  Options is usually not required so having it right before the callback allows us to shorten the logic we need to verify that it was provided.